Library/Homebrew/shims/super/make

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
+export MAKE=${HOMEBREW_MAKE:-make}
```

Comment:
It's probably not strictly necessary, but it was a suggestion in the previous PR to make sure that subprocesses call the correct make.